Rakefile

```diff
@@ -3,20 +3,38 @@ task :dev do
 end
 
 task :prod do
-  sh "bundle exec rackup -p 8080 --env production"
+  sh "bundle exec rackup -o 0.0.0.0 -p 8080 --env production"
```

Comment:
Oh, and since we're using long-form flags in other places, let's use `--host` and `--port` instead of `-o` and `-p` 👍

Reply:
Great idea 💯